dylint/src/driver_builder.rs

```diff
@@ -168,12 +171,37 @@ fn build(opts: &crate::Dylint, toolchain: &str, driver: &Path) -> Result<()> {
     command.success()?;
 
     copy(
-        package
-            .join("target")
-            .join("debug")
-            .join(format!("dylint_driver-{}", toolchain)),
+        package.join("target").join("debug").join(format!(
+            "dylint_driver-{}{}",
+            toolchain,
+            consts::EXE_SUFFIX
+        )),
         driver,
     )?;
 
+    // To succesfully determine the dylint driver Version on Windows,
+    // it is neccesary to place copies of the toolchain dll's next to the driver.
+    #[cfg(target_os = "windows")]
+    {
+        let path = PathBuf::new()
+            .join(env::var(env::RUSTUP_HOME)?)
+            .join("toolchains")
+            .join(toolchain)
+            .join("bin");
+
+        for file in read_dir(path)?.flatten() {
+            let file_name = file.file_name();
+
+            if let Some(file_name) = file_name.to_str() {
+                if file_name.ends_with(".dll") {
```

Comment:
```suggestion
                if file_name.ends_with(consts:: EXE_SUFFIX) {
```
I think a `use` declaration will need to be added.